backend/functions/lib/adapters/gateways/dxtr/orders.dxtr.gateway.ts

```diff
@@ -330,7 +330,7 @@ export class DxtrOrdersGateway implements OrdersGateway {
         CS.CS_DIV + '-' + CS.CASE_ID AS caseId,
         CS.CS_SHORT_TITLE AS caseTitle,
         FORMAT(CS.CS_DATE_FILED, 'yyyy-MM-dd') AS dateFiled,
-        CS.CS_DIV as divisionCode,
```

Comment:
I think because we have `courtDivisionName`, maybe this should be `courtDivisionCode` so that it is very clear that there is a difference.

Reply:
I agree. This is a bit more of a refactor, but I'm cool with that.